content/articles/2018-07-broom-0-5-0.markdown

```diff
@@ -0,0 +1,169 @@
+
+- lme, lme4 and nmle models,
+- brms models,
+- rstanarm models, and 
+- mcmc objects
+
+are one release away from deprecation, and effectively frozen.
+
+## New suggested workflow
+
+When working with many models at the same time, we now recommend using list-columns and a `nest()-map()-unnest()` workflow. This mirrors similar moves across the rest of the tidyverse. We have updated the [kmeans](https://broom.tidyverse.org/articles/kmeans.html), [broom and dplyr](https://broom.tidyverse.org/articles/broom_and_dplyr.html) and  [bootstrapping](https://broom.tidyverse.org/articles/bootstrapping.html) vignettes to reflect the new workflow. Additional, we've updated the bootstrapping vignette to use [rsample](https://tidymodels.github.io/rsample/) rather than the now-deprecated `bootstrap()` function. We no longer recommend the older `group_by()-do()` workflow.
+
+## New vignettes
+
+The list of available tidiers has been moved out of the README and into the [Available Methods](https://broom.tidyverse.org/articles/available-methods.html) vignette. 
+
+We also have two new vignettes that are **strictly works in progress** at the moment. The first covers [Adding New Tidiers](https://broom.tidyverse.org/articles/adding-tidiers.html) and seeks to make the barrier of entry for broom contributions as low as possible. The second contains a [Glossary](https://broom.tidyverse.org/articles/glossary.html) of terms we are developing for use in an upcoming release of broom. This glossary will standardize argument names across tidiers, and column names across tidy output.
+
+## Deprecations
+
+### Deprecations in 0.5.0
+
+- Matrix and vector tidiers have been deprecated in  favor of `tibble::as_tibble()` and `tibble::enframe()`
+- Dataframe tidiers and rowwise dataframe tidiers have been deprecated
+- `bootstrap()` has been deprecated in favor of the [`rsample`](https://tidymodels.github.io/rsample/)
+- `inflate()` has been removed from `broom`
+
+### Soft deprecations
+
+- `sp` tidying methods (in favor of `sf`)
+- `tidy.summaryDefault()` (in favor of `skimr::skim()`)
+- `tidy.table()` (in favor of `tibble::as_tibble()`)
+- Mixed model and bayesian tidiers
+
+## Contributors
+
+Many many thanks to all the following for their thoughtful comments on design, bug reports and PRs! The community of broom contributors has been kind, supportive and insighftul and I look forward to working you all again!
```

Comment:
I'm just one of many names in the list of contributors? 😭

It's probably a good idea to mention in the first paragraph that you've taken over broom as a summer internship (that is, to draw attention to it), in which case I could get a shoutout but we make it clear this is your work. May also be nice to thank RStudio for the internship, and Max and Mara for their assistance with broom and the blog post respectively.